rust/plugin-lib/src/view.rs

```diff
@@ -170,6 +170,36 @@ impl<C: Cache> View<C> {
         self.peer.request_is_pending()
     }
 
+    pub fn add_status_item(&self, key: &str, value: &str, alignment: &str) {
```

Comment:
These three methods should be added to the docs, alongside the other core->frontend methods.